go.mod

```diff
@@ -18,11 +19,13 @@ require (
 	k8s.io/apiextensions-apiserver v0.21.3
 	k8s.io/apimachinery v0.21.3
 	k8s.io/apiserver v0.21.3
+	k8s.io/cli-runtime v0.21.3
 	k8s.io/client-go v0.21.3
 	k8s.io/code-generator v0.21.3
 	k8s.io/component-base v0.21.3
 	k8s.io/component-helpers v0.21.3
 	k8s.io/klog/v2 v2.9.0
+	k8s.io/kubectl v0.21.2
```

Comment:
Not sure `k8s.io/kubectl` could be consumed as a dependency yet, 
but the version should keep consistent(`v0.21.3`) with other k/k components.